### ISSUE TITLE
Prowlarr can't be added as an indexer

### ISSUE THREAD

**Issue by raidenlal** [Radarr/Radarr#6910]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

When adding Prowlarr instance I get this error message:

> Unable to connect to indexer, check the log for more details

Same instance with same configuration works in Sonarr version 3.0.6.1432-ls242 by linuxserver.io

Radarr behaves exactly the same on nightly and develop versions.

### Expected Behavior

Prowlarr instance gets added to Radarr same as it was possible in Sonarr.

### Environment

```markdown
- OS: Docker 20.10.11
- Radarr: 4.0.0.5745-ls44 by linuxserver.io
- Docker Install: yes
- Using Reverse Proxy: yes, nginx, but this is inter-docker communication
- Browser: Chrome Version 96.0.4664.110
```

### What branch are you running?

Develop

### Trace Logs?

I have set trace logs but they seems to be exactly the same verbosity as Info.
```
2022-01-07 01:47:14.7|Warn|Torznab|Unable to connect to indexer

[v4.0.0.5745] System.Xml.XmlException: 'doctype' is an unexpected token. The expected token is 'DOCTYPE'. Line 1, position 3.
   at System.Xml.XmlTextReaderImpl.Throw(Exception e)
   at System.Xml.XmlTextReaderImpl.Throw(String res, String[] args)
   at System.Xml.XmlTextReaderImpl.ThrowUnexpectedToken(String expectedToken1, String expectedToken2)
   at System.Xml.XmlTextReaderImpl.ParseDoctypeDecl()
   at System.Xml.XmlTextReaderImpl.ParseDocumentContent()
   at System.Xml.Linq.XDocument.Load(XmlReader reader, LoadOptions options)
   at System.Xml.Linq.XDocument.Parse(String text, LoadOptions options)
   at NzbDrone.Core.Indexers.Newznab.NewznabCapabilitiesProvider.ParseCapabilities(HttpResponse response) in D:\a\1\s\src\NzbDrone.Core\Indexers\Newznab\NewznabCapabilitiesProvider.cs:line 88
   at NzbDrone.Core.Indexers.Newznab.NewznabCapabilitiesProvider.FetchCapabilities(NewznabSettings indexerSettings) in D:\a\1\s\src\NzbDrone.Core\Indexers\Newznab\NewznabCapabilitiesProvider.cs:line 67
   at NzbDrone.Core.Indexers.Newznab.NewznabCapabilitiesProvider.<>c__DisplayClass4_0.<GetCapabilities>b__0() in D:\a\1\s\src\NzbDrone.Core\Indexers\Newznab\NewznabCapabilitiesProvider.cs:line 34
   at NzbDrone.Common.Cache.Cached`1.Get(String key, Func`1 function, Nullable`1 lifeTime) in D:\a\1\s\src\NzbDrone.Common\Cache\Cached.cs:line 104
   at NzbDrone.Core.Indexers.Newznab.NewznabCapabilitiesProvider.GetCapabilities(NewznabSettings indexerSettings) in D:\a\1\s\src\NzbDrone.Core\Indexers\Newznab\NewznabCapabilitiesProvider.cs:line 34
   at NzbDrone.Core.Indexers.Torznab.Torznab.get_PageSize() in D:\a\1\s\src\NzbDrone.Core\Indexers\Torznab\Torznab.cs:line 23
   at NzbDrone.Core.Indexers.Torznab.Torznab.GetRequestGenerator() in D:\a\1\s\src\NzbDrone.Core\Indexers\Torznab\Torznab.cs:line 27
   at NzbDrone.Core.Indexers.HttpIndexerBase`1.TestConnection() in D:\a\1\s\src\NzbDrone.Core\Indexers\HttpIndexerBase.cs:line 326
RequestUri: http://prowlarr:9696/api?t=caps&apikey=(removed) OK;ContentType: text/html;ContentLength: 1922;ContentSample: <!doctype html><html lang="en"><head><meta charset="utf-8"/><meta name="viewport" content="width=device-width,initial-scale=1"/><meta name="mobile-web-app-capable" content="yes"/><meta name="apple-mobile-web-app-capable" content="yes"/><meta name="theme-color" content="#3a3f51"/><meta name="msapplication-navbutton-color" content="#3a3f51"/><meta name="description" content="Prowlarr (Preview)"/><link rel="apple-touch-icon" sizes="180x180" href="/Content/Images/Icons/apple-touch-icon.png?h=6OFTjFECz9UsB7H/TlShUg"/><link rel="icon" type="image/png" sizes="32x32" href="/Content/Images/Icons/favicon-32x32.png?h=lmqCw7GaBq+Sl0mzceEopQ"/><link rel="icon" type="image/png" sizes="16x16" href="/Content/Images/Icons/favicon-16x16.png?h=eoRdw1mTmIIEK5L0pTH5mw"/><link rel="manifest" href="/Content/Images/Icons/manifest.json?h=DmoSjQqByGr1aINOqvSg9w" crossorigin="use-credentials"/><link rel="mask-icon" href="/Content/Images/Icons/safari-pinned-tab.svg?h=D8kxvQdNZQhWkyStrIiagA" color="#00ccff"/><link rel="shortcut icon" type="image/ico" href="/favicon.ico"/><meta name="msapplication-config" content="/Content/Images/Icons/browserconfig.xml"/><link rel="stylesheet" href="/Content/Fonts/fonts.css?h=mBv1x1/yHnvphLclzeQIBA"><title>Prowlarr (Preview)</title><style>.root {
        overflow: hidden;
        height: 100%; /* needed for proper layout */
      }

      @media only screen and (max-width: 768px) {
        .root {
          display: flex;
          flex-direction: column;
          min-height: 100%;
          height: auto;
        }
      }</style><script defer="defer" src="/vendors.js?h=7Pi2MOrt/+6dLNATKSSUTg"></script><script defer="defer" src="/index.js?h=n6jnR2r2qWJ4pnbgPXo9CQ"></script><link href="/Content/styles.css?h=8yDM9YUaGr1C2lBJ8yWH5Q" rel="stylesheet"></head><body><div id="portal-root"></div><div id="root" class="root"></div></body><script src="/initialize.js"></script></html>

2022-01-07 01:47:14.7|Error|Microsoft.AspNetCore.Diagnostics.ExceptionHandlerMiddleware|An unhandled exception has occurred while executing the request.

[v4.0.0.5745] FluentValidation.ValidationException: Validation failed: 
 -- : Unable to connect to indexer, check the log for more details
   at Radarr.Api.V3.ProviderControllerBase`3.VerifyValidationResult(ValidationResult validationResult, Boolean includeWarnings) in D:\a\1\s\src\Radarr.Api.V3\ProviderControllerBase.cs:line 200
   at Radarr.Api.V3.ProviderControllerBase`3.Test(TProviderDefinition definition, Boolean includeWarnings) in D:\a\1\s\src\Radarr.Api.V3\ProviderControllerBase.cs:line 191
   at Radarr.Api.V3.ProviderControllerBase`3.Test(TProviderResource providerResource) in D:\a\1\s\src\Radarr.Api.V3\ProviderControllerBase.cs:line 140
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.SyncObjectResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeActionMethodAsync()
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeNextActionFilterAsync()
--- End of stack trace from previous location ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()
--- End of stack trace from previous location ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|20_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Radarr.Http.Middleware.BufferingMiddleware.InvokeAsync(HttpContext context) in D:\a\1\s\src\Radarr.Http\Middleware\BufferingMiddleware.cs:line 28
   at Radarr.Http.Middleware.IfModifiedMiddleware.InvokeAsync(HttpContext context) in D:\a\1\s\src\Radarr.Http\Middleware\IfModifiedMiddleware.cs:line 41
   at Radarr.Http.Middleware.CacheHeaderMiddleware.InvokeAsync(HttpContext context) in D:\a\1\s\src\Radarr.Http\Middleware\CacheHeaderMiddleware.cs:line 33
   at Radarr.Http.Middleware.UrlBaseMiddleware.InvokeAsync(HttpContext context) in D:\a\1\s\src\Radarr.Http\Middleware\UrlBaseMiddleware.cs:line 27
   at Radarr.Http.Middleware.VersionMiddleware.InvokeAsync(HttpContext context) in D:\a\1\s\src\Radarr.Http\Middleware\VersionMiddleware.cs:line 28
   at Microsoft.AspNetCore.ResponseCompression.ResponseCompressionMiddleware.InvokeCore(HttpContext context)
   at Microsoft.AspNetCore.Authorization.Policy.AuthorizationMiddlewareResultHandler.HandleAsync(RequestDelegate next, HttpContext context, AuthorizationPolicy policy, PolicyAuthorizationResult authorizeResult)
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.ExceptionHandlerMiddleware.<Invoke>g__Awaited|6_0(ExceptionHandlerMiddleware middleware, HttpContext context, Task task)


2022-01-07 01:47:14.7|Warn|RadarrErrorPipeline|Invalid request Validation failed: 
 -- : Unable to connect to indexer, check the log for more details
```

[AB#2319](https://dev.azure.com/Servarr/7ab38f4e-5a57-4d70-84f4-94dd9bc5d6df/_workitems/edit/2319)

**Comment by Qstick** [comment]
Your Prowlarr url is wrong, you need an indexer id in it 

`http://prowlarr:9696/api?t=caps&apikey=(removed)`

To

`http://prowlarr:9696/23/api?t=caps&apikey=(removed)`

**Comment by raidenlal** [comment]
@Qstick nice that worked, thank you!

However - why /23/api instead of /api? Tooltip says "Path to the api, usually /api". Sonarr does not have this option to change this path. is the 23 a version? If radarr supports this version shouldn't the 23 be implicit?

**Comment by bakerboy448** [comment]
It's the indexer ID

https://wiki.servarr.com/prowlarr/search#api-key-in-query

Sonarr does have that option. Perhaps you're on EOL v2?

https://wiki.servarr.com/sonarr/settings#torrent-tracker-configuration

Though why not add radarr and sonarr to prowlarr and have prowlarr push the indexers as designed?

https://wiki.servarr.com/prowlarr/quick-start-guide#apps

**Comment by raidenlal** [comment]
But which one is 23? Why 23 and not 24? I am not sure how should I discover that 23 by myself for the future.

I use Sonarr v3 (3.0.6.1432-ls242 by linuxserver.io)

I did not know about pushing indexers! Radarr was crying about me using the "all" indexer in jackett and I used it just because I the indexer management is just too cumbersome to be done in jackett and then in radarr. Will try that one:)
